addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1061,7 +1078,7 @@ const ColorpickerUserValueWidget = SelectUserValueWidget.extend({
         const oldColorPalette = this.colorPalette;
         this.colorPalette = new ColorPaletteWidget(this, options);
         if (oldColorPalette) {
-            return this.colorPalette.insertAfter(oldColorPalette.el).then(() => {
+            return this.colorPalette.replace(oldColorPalette.el).then(() => {
```

Comment:
Why? Kinda prefer to let the widget destroy itself entirely by itself.

Reply:
At the beginning of the dev the widget was instantiated while the modal was open which caused the width to shrink due to the scrollbar appearing (2 color_palette is longer than max-width). That was displaying the cursor at the wrong place in the colorpicker.
This is no longer an issue so I will revert that.